pkg/kubelet/kubelet_pods.go

```diff
@@ -1730,9 +1733,10 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 
 	for _, container := range containers {
 		status := &v1.ContainerStatus{
-			Name:  container.Name,
-			Image: container.Image,
-			State: defaultWaitingState,
+			Name:    container.Name,
+			Image:   container.Image,
+			State:   defaultWaitingState,
+			Started: &boolFalse,
```

Comment:
nit:
How about using `k8s.io/util/pointer` here and others?
-> `pointer.Bool(false)`